packages/lib-user/src/components/shared/BarChart/BarChart.js

```diff
@@ -85,14 +105,40 @@ function BarChart({
         {
           property: 'period',
           label: dateRangeLabel.countLabel,
-          render: (period) => {
+          render: ((period, datam, dataIndex) => {
```

Comment:
Very pedantic, but did you mean `datum` and `datumIndex`?

Reply:
I think so! I copied the args from https://v2.grommet.io/datachart#series , but I don't think datam is a thing; datum certainly is!